pom.xml

```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Why was this changed? Apache may have their own specific standard, but that is not much different from the original header before this change. I believe the original header is the default for generating a POM in IntelliJ and should work fine for our project. Having a line for the XML stating the version and encoding is also important to ensure there are zero possible compatibility issues between any IDEs or editors.